Kareem24/books-lib
# add few books to the initial state of the books array

# Test

> Add few books to the initial state

## how to test

> some books should appear after load

> the delete button and submit button when clicked it's functional
